commands/activations.go

```diff
@@ -245,14 +232,57 @@ func printActivationRecord(writer io.Writer, activation whisk.Activation) {
 // RunActivationsList supports the 'activations list' command
 func RunActivationsList(c *CmdConfig) error {
 	argCount := len(c.Args)
+
 	if argCount > 1 {
 		return doctl.NewTooManyArgsErr(c.NS)
 	}
-	output, err := RunServerlessExec(activationList, c, []string{flagCount, flagFull}, []string{flagLimit, flagSkip, flagSince, flagUpto})
+	sls := c.Serverless()
+
+	var name string
+	if argCount > 0 {
+		name = c.Args[0]
+	}
+
+	countFlags, _ := c.Doit.GetBool(c.NS, flagCount)
+	fullFlag, _ := c.Doit.GetBool(c.NS, flagFull)
+	skipFlag, _ := c.Doit.GetInt(c.NS, flagSkip)
+	sinceFlag, _ := c.Doit.GetInt(c.NS, flagSince)
+	upToFlag, _ := c.Doit.GetInt(c.NS, flagUpto)
```

Comment:
These flags are both constructed with `AddStringFlag` and then `GetInt` is used to retrieve their value. Viper seems to handle the type casting, but might be more clear if we changed to using `AddIntFlag`.  E.g. the help text will say `int` instead of `string` which make it more obvious this should be a Unix timestamp and not some other value.